hyp3_rtc_gamma/etc/README_RTC_GAMMA.txt

```diff
@@ -5,20 +5,24 @@ This folder contains radiometric terrain corrected (RTC) products and their asso
 
 Processing Date/Time: [DATE] [TIME] UTC
 
-The folder and each of its contents all share the same base name, using the following convention:  
-S1x_yy_RTzz_aaaaaaaaTbbbbbb_c_def  
-x:        	Sentinel-1 Mission (A or B)  
-yy:       	Beam Mode  
-zz:       	Terrain Correction Resolution  
-aaaaaaaa: 	Start Date of Acquisition (YYYYMMDD)  
-bbbbbb:    	Start Time of Acquisition (HHMMSS)  
-c:        	Processor (Gamma or S1TBX)  
-d:        	gamma-0 (g) or sigma-0 (s) output  
-e:        	amplitude (a) or power (p) output  
-f:        	not filtered (n) or filtered (f)
-
-The source granule used to generate the products contained in this folder is:  
-[GRAN_NAME]  
+The folder and each of its contents all share the same base name, using the following convention:
+S1x_yy_aaaaaaaaTbbbbbb_ppo_RTCzz_G_defkl_ssssss
+x:          Sentinel-1 Mission (A or B)
+yy:         Beam Mode
+aaaaaaaa:   Start Date of Acquisition (YYYYMMDD)
+bbbbbb:     Start Time of Acquisition (HHMMSS)
+pp:         Polarization
+o:          Orbit Type (Precise, Restituted, or Predicted)
```

Comment:
I don't like using `O` for `Predicted`, they don't feel connected to me at all, I'll follow up with Tom.

Reply:
Sounds like, U for Unknown would be a better letter as we're not downloading the Predicted files (unless that's also included in the SAFE and we're pulling it from there)